src/bands/active_bandstructure.h

```diff
@@ -326,7 +332,8 @@ class ActiveBandStructure : public BaseBandStructure {
    */
   static std::tuple<ActiveBandStructure, StatisticsSweep> builder(
       Context &context, HarmonicHamiltonian &h0, Points &points,
-      const bool &withEigenvectors = true, const bool &withVelocities = true);
+      const bool &withEigenvectors = true, const bool &withVelocities = true, 
+      const bool &forceBuildAPP = false);
```

Comment:
We miss the documentation of the parameter forceBuildAPP (rename to forceBuildApp?)

Reply:
I wrote it as forceBuildAPP because it's = forceBuildAsPostprocessing. I could also change to the longer version for clarity, which seems reasonable actually. I can add doc information for that, too.

Reply:
ah, ok, then capitalization makes sense. Just add a line or two of documentation and we are fine.